src/main/resources/com/mathworks/ci/RunMatlabTestsStep/config.jelly

```diff
@@ -0,0 +1,28 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    
+	  <f:entry title="Pdf Report" field="testResultsPdf">
+	        <f:textbox/>
+	  </f:entry> 
+	  
+	  <f:entry title="TAP report" field="testResultsTAP">
+	        <f:textbox/>
+	  </f:entry> 
+	  
+	  <f:entry title="Junit report" field="testResultsJUnit">
+	        <f:textbox/>
+	  </f:entry> 
```

Comment:
Use a consistent capitalization approach. Are these titles displayed in the UI? If so, I'd go with capitalizing them the way the creators do: "PDF", "TAP", "JUnit".

Reply:
No These titles would not be shown in UI . I am using title here just to elaborate what this field is about . I will remove these anyways i feel the field value is descriptive enough .